openslides_backend/action/actions/user/user_scope_permission_check_mixin.py

```diff
@@ -76,6 +76,7 @@ def get_user_scope(
         Returns the scope of the given user id together with the relevant scope id (either meeting, committee or organization).
         """
         meetings: List[int] = []
+        meetingsd: Dict[int, int]
```

Comment:
please use a better name

Reply:
Also, is the declaration here necessary?

Reply:
> please use a better name

done, not quite sure it is better

Reply:
> Also, is the declaration here necessary?

no, removed